pkg/dependencies/factory.go

```diff
@@ -98,8 +94,9 @@ type Dependencies struct {
 	PackageClient               curatedpackages.PackageHandler
 	VSphereValidator            *vsphere.Validator
 	VSphereDefaulter            *vsphere.Defaulter
-	NutanixPrismClient          *v3.Client
+	NutanixClientCache          *nutanix.ClientCache
 	NutanixDefaulter            *nutanix.Defaulter
+	NutanixValidator            *nutanix.Validator
```

Comment:
Where is `NutanixValidator` being used? Also, do we still need `NutanixDefaulter`?

Reply:
It is being generated but I don't see where it is being passed to as a dependency?

Reply:
It's being used by NutanixDatacenterConfig reconciler